Source/Controllers/DataExport/SPExportController.m

```diff
@@ -2851,11 +2851,20 @@ - (void)_tokenizeCustomFilenameTokenField
 
 	NSUInteger start = 0;
 	for(id obj in [exportCustomFilenameTokenField objectValue]) {
+		
+		SPLog(@"obj = %@", obj);
+		
 		NSUInteger length;
 		BOOL isText = NO;
 		if(IS_STRING(obj)) {
 			length = [obj length];
 			isText = YES;
+			
+			// only attempt tokenization if string contains a { or }
+			if([(NSString*)obj containsString:@"{"] == NO && [(NSString*)obj containsString:@"}"] == NO){
```

Comment:
There is a "isString" check, can we convert the object to NSString variable and not cast it multiple times here?

Reply:
sure

Reply:
updated